app/furniture/marketplace/breadcrumbs.rb

```diff
@@ -57,6 +57,11 @@
   link "Add a Delivery Area", marketplace.location(:new, child: :delivery_area)
 end
 
+crumb :edit_delivery_area do |delivery_area|
+  parent :marketplace_delivery_areas, delivery_area.marketplace
+  link "Edit a Delivery Area", marketplace.location(:edit, child: :delivery_area)
```

Comment:
Any chance this could live in `marketplace/config/en.yml`? Something like:

```
en:
  # ...
  marketplace
    delivery_areas: 
      edit: 
        link_to: "Edit Delivery Area '%{name}'"
```

Reply:
will fix in new PR

Reply:
done in https://github.com/zinc-collective/convene/pull/1350